06_Comparing_Object_and_Truth_Tables.ipynb

```diff
@@ -92,7 +94,7 @@
   },
```

Comment:
Remove commented code

Reply:
Thanks for catching that. I removed a couple instances of commented code throughout.